security/pkg/nodeagent/sds/sdsservice.go

```diff
@@ -260,7 +263,15 @@ func (s *sdsservice) StreamSecrets(stream sds.SecretDiscoveryService_StreamSecre
 			defer recycleConnection(conID, resourceName)
 
 			conIDresourceNamePrefix := sdsLogPrefix(conID, resourceName)
-			if !s.skipToken {
+			if s.localJWT {
+				// Running in-process, no need to pass the token from envoy to agent as in-context - use the file
+				tok, err := ioutil.ReadFile("./var/run/secrets/tokens/istio-token")
```

Comment:
"./var/run/secrets/tokens/istio-token" is defined in the constant variables JWTPath and trustworthyJWTPath. Instead of hardcoding the path in the ReadFile(), please refer a constant variable. In addition, the same path should not be defined in multiple constant variables.

Reply:
Missed this, will replace it with the constant.